web-app/HeimScripts/heatmap/preprocess.R

```diff
@@ -0,0 +1,47 @@
+library("WGCNA")
+
+
+main <- function(aggregate=FALSE){
+  msgs = c("")
+  df <- loaded_variables[[length(loaded_variables)]]
```

Comment:
only use the last (subset of) dataset present, in case of multiple nodes and/or subsets?